x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/job_details.js

```diff
@@ -267,7 +267,11 @@ export class JobDetailsUI extends Component {
       });
 
       return (
-        <div className="tab-contents" data-test-subj={`mlJobListRowDetails details-${job.job_id}`}>
+        <div
+          className="tab-contents"
+          css={{ margin: '-8px', padding: '8px' }}
```

Comment:
luckily this isn't actually needed any more.
It was originally added because we wanted to hide the background colour of the outer component and this job details component forced a white background.
However this background colour override has also been removed in this PR.